spec/section/annotations.md

```diff
@@ -1,15 +1,154 @@
 ## Structural Annotations {#annotations}
 
-A [=field=] (`rml:field`) MUST contain zero or more [=structural annotations=]. 
+A [=field=] (`rml:field`) contains zero or more [=structural annotations=].
 <dfn>Structural annotations</dfn> give additional information about the [=field=] (`rml:field`) and its relation to the structure of its [=logical view=] (`rml:LogicalView`).
 
-Following [=structural annotations=] MAY be defined: 
-- unique 
+Following [=structural annotations=] MAY be defined:
```

Comment:
Here we can add immediately the rml terminology for those annotations, e.g. 
- foreign key (`rml:ForeignKeyAnnotation`)
- primary key (`rml:PrimaryKeyAnnotatation`) (i.e. unique and not null) etc.